packages/m2m-gateway/src/api/producerKeychainApiConverter.ts

```diff
@@ -0,0 +1,50 @@
+import { authorizationApi, m2mGatewayApi } from "pagopa-interop-api-clients";
+import { match } from "ts-pattern";
+
+export function toGetProducerKeychainsApiQueryParams(
+  params: m2mGatewayApi.GetProducerKeychainsQueryParams
+): authorizationApi.GetProducerKeychainsQueryParams {
+  return {
+    name: params.name,
+    producerId: params.producerId,
+    limit: params.limit,
+    offset: params.offset,
+    userIds: [],
+  };
```

Comment:
Let's also make explicit the other filter we don't expose

```suggestion
    userIds: [],
    eserviceId: undefined,
  };
```